### ISSUE TITLE
In the typeahead directive, the selected option should be marked with aria-selected="true"

### ISSUE THREAD

**Issue by Codermar** [angular-ui/bootstrap#5233]
In the typeahead directive, the selected option should be marked with aria-selected="true" in addition to marking the aria-activedescendant to its parent. The following code would do that:  (about line 193)

```javascript 
    // Indicate that the specified match is the active (pre-selected) item in the list owned by this typeahead.
    // This attribute is added or removed automatically when the `activeIdx` changes.
    scope.$watch('activeIdx', function(index) {
      if (index < 0) {
        element.removeAttr('aria-activedescendant');
      } else {
          angular.element('#' + popupId + ' li[aria-selected="true"]').attr('aria-selected', false);
          // set the aria-selected attribute
          angular.element('#' + getMatchId(index)).attr('aria-selected', true);
          element.attr('aria-activedescendant', getMatchId(index));
      }
    });
```

**Comment by icfantv** [comment]
@Codermar, the aria items are being tracked in #4772 for the 1.1 release.

**Comment by icfantv** [comment]
I should note that by linking that issue here - it will show up in #4772's feed so, in theory, it shouldn't be missed.  Thanks for the callout.
